packages/frontend/src/api/useDialogs.tsx

```diff
@@ -202,7 +204,7 @@ export const useDialogs = (parties: PartyFieldsFragment[]): UseDialogsOutput =>
     queryKey: [QUERY_KEYS.DIALOGS, mergedPartiesWithSubParties, organizations, location.pathname],
     staleTime: 1000 * 60 * 10,
     retry: 3,
-    queryFn: () => getDialogs(mergedPartiesWithSubParties),
+    queryFn: () => getDialogs(mergedPartiesWithSubParties as string[]),
```

Comment:
Why this casting? I don't think TS complains without? Better to have `string[]` as return type for `flattenParties`?